tests/python/frontend/tflite/test_forward.py

```diff
@@ -650,6 +693,51 @@ def test_all_resize():
         _test_resize(tf.image.resize_nearest_neighbor, data, align_corners=False)
 
 
+#######################################################################
+# Range
+# -------------
+def test_range():
```

Comment:
test_range and test_shape is not invoked from __main__

Reply:
added...